wheels/social-auth-core/meta.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: social-auth-core
+version: 3.1.0+gx0
+src:
+    - https://github.com/python-social-auth/social-core/archive/d3d9c81bdc8b873ff5610e853dc7e9c8f1133e35.zip
+ 
```

Comment:
Build is failing, maybe you need something like https://github.com/galaxyproject/starforge/blob/master/wheels/build/wheels.yml#L179